app/views/api/v2/import_puppetclasses/create.json.rabl

```diff
@@ -0,0 +1,3 @@
+object @environment => nil
```

Comment:
I'm trying to work out if this file is used anywhere?  ditto for "update".  I think only "show" and "index" are used directly, as per the `rabl_template` variable in app/controllers/concerns/api/import_puppetclasses_common_controller.rb.